src/common/types/store-data/visualization-scan-result-data.ts

```diff
@@ -33,12 +33,14 @@ export type TabStopRequirementInstance = {
     html?: string;
 };
 
+export type SingleTabStopRequirementState = {
```

Comment:
nit: personally don't love this name, maybe TabStopRequirementStateInstance?

Reply:
Neither do I, to be honest.

But not sure how I feel about `StateInstance` either ... since the only instance-like thing about it is that it's an instanced object ... and could be confused with what we have generally referred to as instances before as well.

I was thinking re-naming the original type might be best but did not want to do that in this PR since it would get quite large.

How about:

`TabStopRequirementState` to `TabStopTestState` and using `TabStopRequirementState` for this situation?